attributes/default.rb

```diff
@@ -18,6 +18,9 @@
                                  else 'upstart'
                                  end
 
+# Sysconfig directory
+default['mesos']['sysconfig']['directory']          = '/etc/sysconfig'
```

Comment:
wouldn't matter. Creating it is okay. /etc/default is just a adaption of /etc/sysconfig.
For sure we could add a clause here to write it into /etc/default for ubuntu / debian systems.